tests/nnstreamer_filter_tensorflow2_lite/unittest_filter_tensorflow2_lite.cc

```diff
@@ -233,6 +233,45 @@ TEST (nnstreamerFilterTensorFlow2Lite, floatModelResult)
   g_free (is_float);
 }
 
+/**
+ * @brief Positive case to launch gst pipeline
+ */
+TEST (nnstreamerFilterTensorFlow2Lite, floatModelXNNPACKResult)
+{
+  gchar *pipeline;
```

Comment:
In my humble opinion, `g_autofree` is useful for this case.
```cpp
g_autofree gchar *pipeline = g_strdup_printf(...);
```

Reply:
Well I think manually handling the string seems ok.. Can you explain what is the pros on this case?